src/common/components/modal/modal.module.scss

```diff
@@ -67,7 +68,7 @@ $modalPadding: 2rem;
   padding: 0 $modalPadding;
 
   @include respond-above(sm) {
-    padding: 0 $modalPadding;
+    padding: 0 $modalPaddingLarge $basePadding $modalPaddingLarge;
```

Comment:
I know it will fit the design pixel-perfect but dont
We need to keep modal consistent. 
Old padding have been approved by designer and it will stay forever

Reply:
Reverted.